pkgs/development/libraries/libextractor/default.nix

```diff
@@ -1,8 +1,19 @@
-{ fetchurl, stdenv, libtool, gettext, zlib, bzip2, flac, libvorbis
+{ fetchurl, stdenv, substituteAll
+, libtool, gettext, zlib, bzip2, flac, libvorbis
 , exiv2, libgsf, rpm, pkgconfig, fetchpatch
+, gstreamerSupport ? true, gst_all_1 ? null
```

Comment:
@jorsn regarding your concern for closure size increase, is there a feature missing from an end user package because libextractor isn't compile with gstreamer support? If not, I think you should set this to `false` by default.

Reply:
Libextractor includes a cli tool `extract` which I am using e.g. for generating git-annex metadata. I would consider this an end user package. Would you, @doronbehar?
Without gstreamer, there is e.g. no support for id3 tags in mp3 files and worse flac tag support.

Reply:
Ok, I'll leave it `true` and add a comment.